crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -115,8 +115,8 @@ def start_step(step_name)
           raise Crowbar::Error::StartStepExistenceError.new(step_name)
         end
         if running?
-          @logger.warn("Some step is already running.")
-          raise Crowbar::Error::StartStepRunningError.new
+          @logger.warn("Step #{current_step} is already running.")
+          raise Crowbar::Error::StartStepRunningError.new(step_name = current_step)
```

Comment:
@skazi0 yes, this is correct. This check is only done when we try to start an step. In case that we try to start an stedp while another one is already on running state we dont care which one is actually running, as there should only be one step running at the time.